azure-mgmt-cosmosdb/src/main/java/com/microsoft/azure/management/cosmosdb/CosmosDBAccount.java

```diff
@@ -614,5 +675,35 @@
              */
             WithOptionals withDisableKeyBaseMetadataWriteAccess(boolean disabled);
         }
+
+        /**
+         * The stage of the cosmos db update allowing to specify private endpoint connection.
+         */
+        interface WithPrivateEndpointConnection {
+            /**
+             * Start the definition of a private endpoint connection to be attached
+             * to the cosmos db account
+             *
+             * @param name the reference name for the private endpoint connection
+             * @return the first stage of a private endpoint connection definition
+             */
+            PrivateEndpointConnection.UpdateDefinitionStages.Blank<WithOptionals> defineNewPrivateEndpointConnection(String name);
```

Comment:
do we need "New"?

Reply:
like VirtualMachine has defineNewExtension